# **Research**

My current research primarily focuses on understanding the
organization, distribution, interconnection, and comparison of
information in Biological Systems. At the moment, I am trying to
address the following questions:

### **Can we Develop computational algorithms to address the complexities of genome data?**  
We are currently studying different ways to correct Transcriptional
Profiling estimation errors due to the presence of repetitive
sequences in Eukaryotic genomes. We are also developing computational
algorithms to identify different regions present in highly related
genomes.

### **Can Digitalization of Genomic Information Simplify Genomic Analysis?**  
We are particularly intrigued by the prospect of simplifying (i.e.,
digitizing) the information present in DNA, RNA, and Proteins so as to
simplify its manipulation and analysis. We are currently developing
novel computational pipelines dedicated to detecting sequence
variations within related genomes. We think that digitizing emerging
genomic data will not only enable us to use this data effectively but
also to integrate it into Artificial Intelligence, Data Clustering,
and Image Recognition Algorithms, in ways not done before. We posit
that this process of converting biological features into digital
equivalents has the potential to simplify genomic information, making
it easier to uncover previously unnoticed patterns through complex
computational comparisons. This approach has already yielded promising
results by revealing unexpected informational patterns across various
organisms' chromosomes. We believe that it will streamline and enhance
our ability to comprehend different cellular and organismal
states. Moreover, it holds significant promise in revolutionizing our
understanding of diseases, particularly Cancer and Metagenomics. This
informational perspective also contributes to our comprehension of
genome evolution, especially in the field of comparative genomics and
microbial metagenomics.

### **Do Non-Coding RNAs regulate the interaction of Ribosomal RNAs and Ribosomal proteins?**  
Using _Homo sapiens_ Next Generation Sequencing RNA data, we have
recently detected the presence of a set on non-coding RNAs with
homology to Ribosomal RNAs. We hypothesize that this set of non-coding
RNAs interact with segments of Ribosomal RNAs so as to participate (by
interfering or enhancing), the interaction of ribosomal proteins with
the Ribosomal RNAs. If confirmed, this observation would reveal yet
another level of translational regulation. It would also assign
function to a large set of non-coding RNAs. In our view, this is a
very important, and yet undisclosed, observation that has the
potential of changing our current view of translational regulation in
all organisms.

### **How are Sequence Variations Detected and Targeted for Silencing in Fungal Meiosis?**  
We employ the filamentous fungus _Neurospora crassa_ as a model
organism to uncover and comprehend the intricate molecular components
responsible for sequence-based comparisons between homologous
chromosomes, leading to the initiation of Meiotic Silencing, a
phenomenon driven by RNA-mediated processes. Currently, our primary
focus centers on the exploration of whether genes recognized for their
significance in Meiotic Transvection/Silencing also contribute to the
occurrence of Repeat Induced Point Mutation (RIP) phenomena.
